src/locale/ku.js

```diff
@@ -18,6 +18,7 @@ const locale = {
     LLL: 'D MMMM YYYY HH:mm',
     LLLL: 'dddd, D MMMM YYYY HH:mm'
   },
+  meridiem: hour => (hour < 12 ? 'پ.ن' : 'د.ن'),
```

Comment:
Hi, should we keep the same with momentjs as https://github.com/moment/moment/blob/e96809208c9d1b1bbe22d605e76985770024de42/locale/ku.js#L76
```
meridiem: function (hour, minute, isLower) {
            if (hour < 12) {
                return 'به‌یانی';
            } else {
                return 'ئێواره‌';
            }
        },
```

Reply:
Hi @iamkun 

Not sure if this was a question.

The rationale behind using `پ.ن` instead of `بەیانی` is that, it's short version of it, just as `AM`.
Usually it's a preferred way of using those post-fixes with time in Kurdish language, not sure if we use `بەیانی` and `ئێوارە` whatsoever.

Reply:
alright, thanks for this explanation